### PR TITLE
Add link to donate directly to Street Support #421

### PR DESCRIPTION
**Pull request by mikecoram**

### CODE REVIEW

File: src/pages/give-help/business-support/index.hbs [hunk 1]
```diff
@@ -57,6 +57,10 @@ page: partnership
       <li>£18k to provide a years employment at Street Support for a GROW trainee (someone experiencing homelessness).</li>
     </ul> 
 
+    <a class="btn btn--brand-e" href="https://www.justgiving.com/streetsupportnetwork">
+      <span class="btn__text">Donate to StreetSupport</span>
```
Comment (vincelee888) [inline src/pages/give-help/business-support/index.hbs]: can we make this "Street Support", with a space

File: src/pages/give-help/business-support/index.hbs [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
-title: Partnership - Street Support
+title: Business Support - Street Support
 description: Street support makes it easier to help people facing homelessness. Here's how to get in touch.
 image: https://streetsupport.net/assets/img/og/street-support.png
 layout: default.hbs
 permalink: false
 jsBundle: sponsor
 section: givehelp
-page: partnership
+page: business-support
```
Comment (vincelee888) [inline src/pages/give-help/business-support/index.hbs]: leave this as "partnership"

this field is used by metalsmith to inject a class into the body tag. this is then picked up by the css to set the active nav item

Comment (mikecoram) [reply src/pages/give-help/business-support/index.hbs]: I see, I've reverted the changes.

Comment (mikecoram) [reply src/pages/give-help/business-support/index.hbs]: https://github.com/mikecoram/streetsupport-web/commit/6d67387d6d39f3e29f16ecca341edb971d54d845

https://github.com/mikecoram/streetsupport-web/commit/8d6fd00dfdf0dd8f4d421fce201d2446a537e9a7

File: src/pages/give-help/business-support/index.hbs (hunk 1 above)
Comment (mikecoram) [reply src/pages/give-help/business-support/index.hbs]: Done! 58cbff7dd30a5cba0421cff2299e284f49b41ed0

**Pull request merged**